### PR TITLE
Names refactor & code cleanup

### PR DESCRIPTION
**Pull request by abalakh**

### CODE REVIEW

File: airgun/widgets.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ class Search(Widget):
     default_result_locator = Text("//a[contains(., '%s')]")
 
     def fill(self, value):
-        self.search_field.fill(value)
+        return self.search_field.fill(value)
```
Comment (oshtaier) [inline airgun/widgets.py:74]: `fill` is never expected to return something. Can you explain reason for that change?

File: airgun/widgets.py [hunk 2]
```diff
@@ -196,3 +196,15 @@ def current_loc(self):
...
+    def read(self):
+        """As reading organization and location is not atomic operation: needs
+        mouse moves, clicks, etc, and this widget is included in every view -
+        calling meth:`airgun.views.common.BaseLoggedInView.read` for any view
+        will trigger reading values of :class:`ContextSelector`. Thus, to avoid
+        significant performance degradation it should not be readable.
+
+        Use :meth:`current_org` and :meth:`current_loc` instead.
+        """
+
```
Comment (oshtaier) [inline airgun/widgets.py]: redundant line

File: airgun/widgets.py (hunk 1 above)
Comment (abalakh) [reply airgun/widgets.py:74]: Actually `fill` is always expected to return a value. It should be `bool` whether something (at least one of fields) was filled or not. That's probably useful to determine whether there was a point in calling `fill` or all the fields had expected values already. Will we use that somewhere or not - hard to tell at this point, but at least let's follow default interface.

**Comment by abalakh** [comment]
@oshtaier addressed.

**Review by oshtaier — APPROVED** [review]

File: airgun/browser.py [hunk 3]
```diff
@@ -479,6 +479,18 @@ class AirgunBrowserPlugin(DefaultPlugin):
...
+        """Invoked before clicking on an element. Ensure page is fully loaded
+        before clicking.
+        """
+        self.ensure_page_safe()
+
+    def after_click(self, element):
+        """Invoked after clicking on an element. Ensure page is fully loaded
+        before proceeding further.
+        """
+        self.ensure_page_safe()
```
Comment (mfalesni) [inline airgun/browser.py:492]: Click does this by default.

**Pull request merged**